adapter/stdioLogger/config/BUILD

```diff
@@ -6,7 +6,7 @@ gogoslick_proto_library(
         "gogoproto/gogo.proto": "github.com/gogo/protobuf/gogoproto",
     },
     imports = [
-        "../../../../external/com_github_gogo_protobuf",
+        "external/com_github_gogo_protobuf",
```

Comment:
changes like this should be in a separate PR to focus attention on the actual server.